CONTRIBUTING.md

```diff
@@ -1,19 +1,84 @@
-# Contribution Guidelines
-## Eligibility
+<h1 align="center">Contribution Guidelines</h1>
+
+---
+
+<h2 align="center">Eligibility</h2>
+
 A project is eligible to be added to this list if it is:
-- **awesome** :sunglasses:
-> Research if the stuff you're including is actually awesome. Only put stuff on the list that you or another contributor can personally recommend. You should rather leave stuff out than include too much.
-- well documented (wiki, javadocs or command line help)
-- open source with a [free license](https://www.gnu.org/licenses/license-list.html)
-- maintained (no dead projects)
+- **Awesome!** :sunglasses:
+    - Research if the stuff you're including is actually awesome.
+    - Only put stuff on the list that you or another contributor can personally recommend.
+    - You should rather leave stuff out than include too much.
+- Well-documented (wiki, descriptive README.md, javadocs or command-line help available).
+- Open-source with a [free license](https://www.gnu.org/licenses/license-list.html).
+- Actively maintained (no dead projects!).
+
+---
 
-## Adding a new entry to the list
+<h2 align="center">Entry Additions</h2>
 
 1. Make sure the entry is [eligible](#eligibility) for this list.
-2. Click on the pen icon at the top of the README file.
-3. Add the entry at the alphabetically right position. It should use the following convention with a [SPDX License Identifier](https://spdx.org/licenses):
-```markdown
-[Project Name](Source link) - Description. ([Wiki](Wiki Link)) `SPDX License Identifier`
-```
-5. At the bottom of the page state what you added.
+2. Click on the pen icon at the top of the [README](README.md).
+3. Add the entry such that it respects alphabetical order within its parent category. It should use the following convention with a [SPDX License Identifier](https://spdx.org/licenses):
+
+<div align="center">
+  <table>
+    <tr>
+      <th>Wiki Available</th> <th>Markdown Format</th>
+    </tr>
+    <tr>
+      <td align="center">✔️</td>
+      <td align="center">
+        <br />
+        <pre lang="markdown">[Project Name](Source Link) - Description. ([Wiki](Wiki Link)) `SPDX License Identifier`</pre>
+      </td>
+    </tr>
+    <tr>
+      <td align="center">❌</td>
+      <td align="center">
+        <br />
+        <pre lang="markdown">[Project Name](Source Link) - Description. `SPDX License Identifier`</pre>
+      </td>
+    </tr>
+  </table>
+</div>
```

Comment:
I think this kind of table formatting is covered by GitHub flavored Markdown.

Wherever possible we should stick to Markdown instead of HTML.

Reply:
GFM will not allow the code blocks to be present the way they are here.

Reply:
Have just confirmed: https://stackoverflow.com/a/46648510